fe/fe-core/src/main/java/org/apache/doris/backup/AbstractJob.java

```diff
@@ -87,7 +87,7 @@ protected AbstractJob(JobType type, String label, long dbId, String dbName,
         this.label = label;
         this.dbId = dbId;
         this.dbName = dbName;
-        this.createTime = System.currentTimeMillis();
```

Comment:
I think `this.` in constructor can remain.
Because I think it's better to keep same style.